### DISCUSSION TITLE
[FEATURE] Integration with Cirrus CI

### DISCUSSION THREAD

**Discussion by confused-Techie** [Infisical/infisical#5162] [Feature Request]
### Feature description

Being able to integrate closer with Cirrus CI as a build platform and retrieve secrets from it would be amazing, and super helpful for those utilizing that platform. 

### Why would it be useful?

For those on the platform, being able to import secrets, especially authentication tokens or signing keys would be very helpful to keep the configuration all in one place.

### Additional context

While I don't yet know the full scope of this issue I'd be more than happy to help out wherever possible! (Especially if on the JavaScript side of things!)

**Comment by dangtony98** [comment]
Hey @confused-Techie!

This is cool! I haven't used Cirrus CI before but it'd be a great addition to Infisical's growing list of integrations 😄

I've added this to our [roadmap](https://www.notion.so/infisical/be2d2585a6694e40889b03aef96ea36b?v=5b19a8127d1a4060b54769567a8785fa).

If you want, feel free to take a stab at it! The integrations code for Infisical is very flexible and easy to work with, so I'm sure it's do-able. 

Also check out our community [Slack](https://infisical.com/slack) if you haven't yet — it's best for communicating/coordinating.

**Comment by confused-Techie** [comment]
Fantastic! Thanks for adding it.

And I'll have to find some time to check it out when I can, but good to know where I can turn in case I have any questions

**Comment by yadavshubham01** [comment]
@dangtony98 @vmatsiiako Hi! Is this issue still available? I’d like to take it up and implement the Cirrus CI integration. assign me?

**Comment by akhilmhdh** [comment]
We’re reorganising our GitHub issues list. All feature requests will now be moved to the [feature request discussions (https://github.com/Infisical/infisical/discussions/categories/feature-request)] and users can upvote them to prioritise. This way, we can select features as our list empties.

Moving this to discussion.
